corehq/apps/hqadmin/views/users.py

```diff
@@ -166,7 +166,7 @@ def get(self, request, *args, **kwargs):
         if not domain.endswith(settings.HQ_ACCOUNT_ROOT):
             full_username = format_username(username, domain)
 
-        self.user = CommCareUser.get_by_username(full_username)
+        self.user = CouchUser.get_by_username(full_username)
```

Comment:
I don't think this'll work - the code above expects usernames to be of the format `<username>@<domain>.commcarehq.org`, and would butcher a normal username/email.  Have you been able to test it with a web user with a normal email address?

Reply:
okay I added some logic to handle this issue.